lib/views/multiline.coffee

```diff
@@ -0,0 +1,21 @@
+class MessageLine extends HTMLElement
```

Comment:
Because I prefer to use `new MessageLine.setText text` instead of fumbling around with `document.createElement 'div'; ...`. I could use a `div.message-line`, but that seems like more code with no apparent benefit.